evm_loader/lib/src/types/indexer_db.rs

```diff
@@ -1,176 +0,0 @@
-use {
-    super::{do_connect, ChDbConfig, PgError, PgResult, TxParams},
-    ethnum::U256,
-    evm_loader::types::Address,
-    solana_sdk::clock::Slot,
-    std::{
-        convert::{TryFrom, TryInto},
-        sync::Arc,
-    },
-    tokio_postgres::Client,
-};
-
-#[derive(Debug, Clone)]
-pub struct IndexerDb {
-    pub client: Arc<Client>,
-}
-
-const TXPARAMS_FIELDS: &str =
-    "from_addr, COALESCE(to_addr, contract), calldata, value, gas_limit, nonce";
-
-impl IndexerDb {
-    pub async fn new(config: &ChDbConfig) -> Self {
-        let client = do_connect(
-            &config.indexer_host,
-            &config.indexer_port,
-            &config.indexer_database,
-            &config.indexer_user,
-            &config.indexer_password,
```

Comment:
These fields can also be deleted from struct `ChDbConfig`.

Their corresponding env vars (`NEON_DB_INDEXER_HOST`, `NEON_DB_INDEXER_PORT`, `NEON_DB_INDEXER_DATABASE`, `NEON_DB_INDEXER_USER`, `NEON_DB_INDEXER_PASSWORD`) can be deleted too.

Also the values provided for these env vars from `docker-compose-ci.yml` and `INDEXER_DB_PASSWORD: ${{secrets.INDEXER_DB_PASSWORD}}` from `pipeline.yml` can be deleted too.

Reply:
Deleted